src/generator.rs

```diff
@@ -16,7 +15,7 @@ pub fn generate_crs() -> Result<Parameters<Bls12>, SynthesisError> {
         params,
         sk: None,
         vrf_input: None,
-        auth_path: vec![(Fr::random(rng), PathDirection::random(rng)); 10],
+        auth_path: vec![None; 10],
```

Comment:
I do not understand either the old random value here or the change.  I suppose the old value was simply because no canonical input like `None` exists and the CRS generation does not require the actual data?

Reply:
Yeah. I think it's because we don't need the input when generating the CRS.